classes/mcPy/MultiProcessing.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import multiprocessing
+import time
+from queue import Full
+
+import classes.Server as Server
+
+from ..Exceptions import ServerException
+
+
+class MultiProcessing():
+
+    def __init__(self, server: Server, worker_number, max_size=100000):
+        self.server = server
+        self.worker_number = worker_number
+        self.max_size = max_size
+        self.started = True
+        self.TASK_LIST = multiprocessing.Queue(maxsize=max_size)
+        self.workers = []
+        for i in range(self.worker_number):
+            func_args = (self.TASK_LIST,)
+            p = multiprocessing.Process(target=MultiProcessing.worker, args=func_args)
+            self.workers.append(p)
+
+    def start(self):
+        self.started = True
+        for p in self.workers:
+            logging.info('Starting worker')
+            p.start()
+            logging.info('Worker started')
```

Comment:
maybe numerate the workers or something, it can be a bit confusing to see workers start up multiple times.

Reply:
Done